eth2util/network.go

```diff
@@ -59,12 +61,22 @@ var (
 	}
 )
 
-var supportedNetworks = []Network{
-	Mainnet, Goerli, Gnosis, Sepolia, Holesky,
+var (
+	addNetworkOnce    sync.Once
```

Comment:
nit: can remove this if `supportedNetworks` is written only once on startup, then only read

wdyt?

Reply:
agreed, removed sync once and updated the godoc.